source/class/qx/ui/form/FileSelectorButton.js

```diff
@@ -136,7 +140,7 @@ qx.Class.define("qx.ui.form.FileSelectorButton", {
       let label = new qx.html.Element("label", {}, { for: id });
       label.addListenerOnce(
         "appear",
-        function (e) {
+        e => {
           label.add(input);
         },
         this
```

Comment:
Specifying a binding for an arrow function doesn't work. This function doesn't use `this`, so the arrow function is fine, but line 142 specifying a binding is probably inappropriate. I'll approve, but I suggest removing that third argument to `addListenerOnce`.

Reply:
so true